controllers/blog.js

```diff
@@ -0,0 +1,65 @@
+const articleModel = require('../models/blog')
+
+const createNewArticle = (req, res) => {
+  const newBlog = new articleModel({
+    title: 'Buy the PS4 While You still can',
+    author: 'Yvad Titi',
+    content: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum'
+  })
+  newBlog.save()
+    .then((result) => {
+      res.send(result)
+    })
+    .catch((err) => {
+      console.log(err)
+    })
+}
+
+
+const getAllArticles = (req, res) => {
```

Comment:
Try to use async-await rather than the chaining of .then promises

Reply:
working on it 
thanks